components/edit-func.js

```diff
@@ -0,0 +1,133 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import dynamic from 'next/dynamic'
+
+import { LinkButton } from './button'
+import { canEdit } from '../libs/general'
+
+/*eslint no-unused-vars: ["error", { "varsIgnorePattern": "^_.*" }]*/
```

Comment:
This fixes unused variables in `filterItem`:
```
const filterItem = (item) => {
    const {
        ['@type']: _type,
        ['@id']: _id,
        ['@context']: _context,
        ['uuid']: _uuid,
        ['actions']: _actions,
        ['access_keys']: _access,
        ['schema_version']: _version,
        ['title']: _title,
        ...filtered } = item
    return filtered
}
```

I don't know, it was giving me trouble, so I got it to ignore unused variable lint errors on vars prepended with underscore.

Reply:
Oh, I see the error in edit.js. But it’s not needed in _this_ file, is it?

Reply:
Oh, dang, I remember now. I completely forgot that _I_ copied that code. Yeah, the ref contains a reference to the button DOM element. It’s puzzling-looking here because we don’t actually see ref used, but the `<Link>` component uses it internally. So yeah, you have it right.